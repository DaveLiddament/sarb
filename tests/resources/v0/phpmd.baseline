{"historyAnalyser":"git","historyMarker":"0f078b7d445f661991605b91d8ddb9c7f796242d","resultsParser":"phpmd-json","analysisResults":[{"lineNumber":28,"fileName":"static-analysis-baseliner\/src\/Domain\/Analyser\/BaseLineResultsRemover.php","type":"LongVariable","message":"Avoid excessively long variable names like $latestAnalysisResults. Keep variable name length under 20.","fullDetails":"{\"fileName\":\"\\\/vagrant\\\/static-analysis-baseliner\\\/src\\\/Domain\\\/Analyser\\\/BaseLineResultsRemover.php\",\"violation\":{\"beginLine\":28,\"endLine\":28,\"package\":null,\"function\":null,\"class\":null,\"method\":null,\"description\":\"Avoid excessively long variable names like $latestAnalysisResults. Keep variable name length under 20.\",\"rule\":\"LongVariable\",\"ruleSet\":\"Naming Rules\",\"externalInfoUrl\":\"https:\\\/\\\/phpmd.org\\\/rules\\\/naming.html#longvariable\",\"priority\":3}}"},{"lineNumber":30,"fileName":"static-analysis-baseliner\/src\/Domain\/Analyser\/BaseLineResultsRemover.php","type":"LongVariable","message":"Avoid excessively long variable names like $baseLineAnalysisResults. Keep variable name length under 20.","fullDetails":"{\"fileName\":\"\\\/vagrant\\\/static-analysis-baseliner\\\/src\\\/Domain\\\/Analyser\\\/BaseLineResultsRemover.php\",\"violation\":{\"beginLine\":30,\"endLine\":30,\"package\":null,\"function\":null,\"class\":null,\"method\":null,\"description\":\"Avoid excessively long variable names like $baseLineAnalysisResults. Keep variable name length under 20.\",\"rule\":\"LongVariable\",\"ruleSet\":\"Naming Rules\",\"externalInfoUrl\":\"https:\\\/\\\/phpmd.org\\\/rules\\\/naming.html#longvariable\",\"priority\":3}}"},{"lineNumber":36,"fileName":"static-analysis-baseliner\/src\/Domain\/Analyser\/internal\/BaseLineResultsComparator.php","type":"LongVariable","message":"Avoid excessively long variable names like $baseLineAnalysisResults. Keep variable name length under 20.","fullDetails":"{\"fileName\":\"\\\/vagrant\\\/static-analysis-baseliner\\\/src\\\/Domain\\\/Analyser\\\/internal\\\/BaseLineResultsComparator.php\",\"violation\":{\"beginLine\":36,\"endLine\":36,\"package\":null,\"function\":null,\"class\":null,\"method\":null,\"description\":\"Avoid excessively long variable names like $baseLineAnalysisResults. Keep variable name length under 20.\",\"rule\":\"LongVariable\",\"ruleSet\":\"Naming Rules\",\"externalInfoUrl\":\"https:\\\/\\\/phpmd.org\\\/rules\\\/naming.html#longvariable\",\"priority\":3}}"}]}