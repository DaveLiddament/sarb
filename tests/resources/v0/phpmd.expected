{
    "historyAnalyser": "git",
    "historyMarker": "0f078b7d445f661991605b91d8ddb9c7f796242d",
    "resultsParser": "phpmd-json",
    "analysisResults": [
        {
            "lineNumber": 28,
            "fileName": "static-analysis-baseliner\/src\/Domain\/Analyser\/BaseLineResultsRemover.php",
            "type": "LongVariable",
            "message": "Avoid excessively long variable names like $latestAnalysisResults. Keep variable name length under 20.",
            "severity": "error"
        },
        {
            "lineNumber": 30,
            "fileName": "static-analysis-baseliner\/src\/Domain\/Analyser\/BaseLineResultsRemover.php",
            "type": "LongVariable",
            "message": "Avoid excessively long variable names like $baseLineAnalysisResults. Keep variable name length under 20.",
            "severity": "error"
        },
        {
            "lineNumber": 36,
            "fileName": "static-analysis-baseliner\/src\/Domain\/Analyser\/internal\/BaseLineResultsComparator.php",
            "type": "LongVariable",
            "message": "Avoid excessively long variable names like $baseLineAnalysisResults. Keep variable name length under 20.",
            "severity": "error"
        }
    ]
}
